Source/Fuse.Controls.Native/DarkMode.uno

```diff
@@ -66,20 +70,59 @@ namespace Fuse.Controls.Native.iOS
 		{
 			if (_instance != null) return;
 			Resource.SetGlobalKey(_instance = this, "FuseJS/DarkMode");
+
+			SetupAndroidListener();
```

Comment:
Here we can do
```
if defined(ANDROID)
    SetupAndroidListener();
```

After that we can drop the following dummy variants of this method for `iOS` and `!MOBILE`.

Reply:
Ok cool...